### PR TITLE
Added images for IAM and cleared up needed steps

### PR DESCRIPTION
**Pull request by wombat**

### CODE REVIEW

File: prereqs.adoc [hunk 1]
```diff
@@ -22,26 +22,36 @@ on your machine.
...
+
+image::images/add-policies-to-iam-user.png[]
+
+Check all five and hit next.
 
 Please review this link:https://github.com/kubernetes/kops/blob/master/docs/aws.md#setup-iam-user[link]
 for additional info on IAM permissions
 
-== AWS Region and Availability Zones
+== Configure the AWS CLI
```
Comment (arun-gupta) [inline prereqs.adoc]: there are possibly some broken links because of changed header. Can you make sure it all looks good?

Comment (wombat) [reply prereqs.adoc]: I have searched for the old header but couldn´t find any results

File: prereqs.adoc [hunk 2]
```diff
@@ -22,26 +22,36 @@ on your machine.
...
-If you already have an AWS Account, you need to create an IAM user and attach these http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html[IAM policies].
+If you already have an AWS Account, you need to create an IAM user here: https://console.aws.amazon.com/iam/home?#/users$new
+Just give it a useful name (e.g. `workshop`) and make sure you check the `Programmatic access` box:
 
-    AmazonEC2FullAccess
-    AmazonRoute53FullAccess
-    AmazonS3FullAccess
-    IAMFullAccess
-    AmazonVPCFullAccess
+image::images/add-iam-user.png[]
```
Comment (arun-gupta) [inline prereqs.adoc]: Can you also add the CLI version of assigning these perms?

Comment (wombat) [reply prereqs.adoc]: What do you mean by that?

Comment (arun-gupta) [reply prereqs.adoc]: These permissions can also be specified using AWS CLI. Can you include that as well?

Comment (cloudymind) [reply prereqs.adoc]: you're talking about these cli calls, right?
https://github.com/kubernetes/kops/blob/master/docs/aws.md#setup-iam-user

Comment (cloudymind) [reply prereqs.adoc]: maybe we should push all of the images to the setup-iam-user page and just keep the original page to point to it

Comment (arun-gupta) [reply prereqs.adoc]: Want to make sure the perms are captured and shown using AWS CLI as that is a preferred approach by a lot of developers.

Comment (wombat) [reply prereqs.adoc]: ok - will update it now. Will basically copy/paste the ones from kops and adjust names, etc

Comment (arun-gupta) [reply prereqs.adoc]: LMK when PR is updated

**Comment by wombat** [comment]
@arun-gupta switched to cli based iam setup

**Comment by cloudymind** [comment]
did we lose $AWS_DEFAULT_REGION? that variable is no longer exported.

**Comment by wombat** [comment]
@cloudymind Do we need it later on? aws cli is configured to use a default region

**Comment by cloudymind** [comment]
i haven't looked at all the places it's referenced but i saw it on the kops cluster install steps

**Comment by arun-gupta** [comment]
I thought that was getting set with `aws configure` but apparently not.

@wombat can you send another PR by requiring to set `$AWS_DEFAULT_REGION`?

**Comment by wombat** [comment]
@cloudymind @arun-gupta My idea was to remove the variable in the other sections as it is not needed

**Comment by cloudymind** [comment]
that would probably be the other way to go. it is being set with aws configure so the later references are just redundant

**Comment by wombat** [comment]
I haven´t finished testing the other sections yet but it is on my todo list

**Comment by arun-gupta** [comment]
Removed references to `$AWS_DEFAULT_REGION` from 3 chapters.

**Pull request merged**